duckduckgo/macos-browser
# Image of the shield with dot replaced

Task/Issue URL: https://app.asana.com/0/1177771139624306/1201667379550414/f

**Description**:
New image asset for the shield with dot

**Steps to test this PR**:
1. Open http://dsl.sk/
1. Make sure there is no border around red dot when mouse is over the shield

Before
<img width="32" alt="before" src="https://user-images.githubusercontent.com/57389842/151966720-a52af7c4-8ace-4161-b244-092014be00ee.png">

After
<img width="31" alt="after" src="https://user-images.githubusercontent.com/57389842/151966727-1bdcec06-46d1-4fd3-af46-49aa247e5923.png">


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
